win32/src/winapi/kernel32/thread.rs

```diff
@@ -101,6 +101,12 @@ pub async fn CreateThread(
     }
 }
 
+#[win32_derive::dllexport]
+pub fn ExitThread(machine: &mut Machine, dwExitCode: u32) {
+    log::info!("ExitThread({}) on cpu {}", dwExitCode, machine.emu.x86.cur_cpu);
+    machine.emu.x86.cpu_mut().state = x86::CPUState::Exit(dwExitCode);
```

Comment:
Can you add a TODO about thread cleanup here, in particular freeing the stack.  (There is other cleanup mentioned in the ExitThread docs)